src/chatbot/src/chatbot/aiml/DefaultSubs.py

```diff
@@ -48,8 +50,8 @@
     "hers": "his",
     "herself": "himself",
 }
-
-defaultPerson = {
+# These now match the comment above
+defaultPerson2 = {

```

Comment:
this looks incorrect. According to the AIML spec, person2 converts from 1st person to second person for example, I <--> you  while the below is clearly for 3rd person ... 

Likewise, the person rule is supposed to convert 1st to 3rd person. 

I'm guessing that the "I like bannanas" rule is broken -- it is using person instead of person2 -- I suggest that the correct fix is to fix the rule, instead of the definition of the conversion.